pkg/operation/botanist/kubeapiserver.go

```diff
@@ -449,12 +450,21 @@ func (b *Botanist) DeployKubeAPIServer(ctx context.Context) error {
 		return fmt.Errorf("secret %q not found", kubeapiserver.SecretNameUserKubeconfig)
 	}
 
+	// add CA bundle as ca.crt to kubeconfig secret for backwards-compatibility
+	caBundleSecret, found := b.SecretsManager.Get(v1beta1constants.SecretNameCACluster)
+	if !found {
+		return fmt.Errorf("secret %q not found", v1beta1constants.SecretNameCACluster)
+	}
+
+	kubeconfigSecretData := userKubeconfigSecret.DeepCopy().Data
+	kubeconfigSecretData[secretutils.DataKeyCertificateCA] = caBundleSecret.Data[secretutils.DataKeyCertificateBundle]
```

Comment:
Same question here

Reply:
I think we should definitely keep it here for backwards-compatibility. 
If we change the key here, then we should rather drop this value at all and tell folks to read from the dedicated secret instead. In this case, they will need to adapt to the breaking change anyway, if they are already reading from this secret.